dowhy/causal_estimators/causalml.py

```diff
@@ -0,0 +1,108 @@
+import inspect
+import numpy as np
+import pandas as pd
+import pdb
+
+from dowhy.causal_estimator import CausalEstimate
+from dowhy.causal_estimator import CausalEstimator
+from importlib import import_module
+import causalml
+
+class Causalml(CausalEstimator):
+
+    def __init__(self, *args, **kwargs):
+
+        super().__init__(*args, **kwargs)
+
+        # Add the core method used in the estimator
+        self.identifier_method = self._target_estimand.identifier_method
+        self.logger.debug("The identifier method used {}".format(self.identifier_method))
+
+        # Check the backdoor variables being used
+        self.logger.debug("Back-door variables used:" +
+                          ",".join(self._target_estimand.backdoor_variables))
+        
+        # Add the observed confounders and one hot encode the categorical variables
+        self._observed_common_causes_names = self._target_estimand.backdoor_variables
+        if self._observed_common_causes_names:
+            # Get the data of the unobserved confounders
+            self._observed_common_causes = self._data[self._observed_common_causes_names]
+            # One hot encode the data if they are cateogorical

```

Comment:
typo in comment. Do check the comments once for typos before opening a pull request.